sdccc-bom/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ javaPlatform {
 dependencies {
     api(enforcedPlatform(libs.org.junit.jupiter.bom))
     api(enforcedPlatform(libs.org.apache.logging.log4j.log4j.bom))
+    api(enforcedPlatform(libs.org.jetbrains.kotlin.kotlin.bom))
```

Comment:
Should also be enforced in the plugin-bom.

Reply:
Done